packages/@coorpacademy-components/src/template/app-player/popin-correction/index.js

```diff
@@ -37,24 +54,42 @@ Question.propTypes = {
   answer: PropTypes.string
 };
 
-const PopinCorrection = props => {
-  const {header = {}, question, resources, klf, tips, onClick} = props;
-  const tabs = extractTabs([resources, klf, tips]);
+class PopinCorrection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {closing: false};
+  }
+
+  render() {
+    const {header = {}, question, resources, klf, tips, onClick} = this.props;
+    const tabs = extractTabs({resources, klf, tips});
+    const isLoading = isNil(header.lives);
+    const delayedHeader = wrapHeaderClick(header, headerClick => {
+      if (!this.state.closing) {
+        this.setState({closing: true}, () => setTimeout(headerClick, ANIMATION_DELAY));
```

Comment:
essaie avec https://jonsuh.com/blog/detect-the-end-of-css-animations-and-transitions-with-javascript/